test/cmdlineargs.jl

```diff
@@ -97,6 +97,12 @@ let exename = `$(Base.julia_cmd()) --startup-file=no`
         @test startswith(read(`$exename --help`, String), header)
     end
 
+    # --project
+    let expanded = abspath(expanduser("~/foo"))
+        @test occursin(expanded, readchomp(`$exename --project='~/foo' -E 'Base.active_project()'`))
+        @test !occursin("~", readchomp(`$exename --project='~/foo' -E 'Base.active_project()'`))
```

Comment:
I believe this will fail on Windows. I think the first test is enough.

Reply:
ah, right